tools/cli/NearObjectCli.cxx

```diff
@@ -9,6 +9,9 @@
 #include <variant>
 #include <vector>
 
+#define MAGIC_ENUM_RANGE_MIN 0
+#define MAGIC_ENUM_RANGE_MAX 256
```

Comment:
This will increase build times since it applies to everything in this file, not just `UwbConfiguration::ParameterTag` where it's needed. You can instead specialize `enum_range<UwbConfiguration::ParameterTag>` in the `customize` namespace.

We should also tighten the maximum range to reduce compile times as much as possible. It looks like the min is `FiraPhyVersion == 0x80` and max is `MaxRrRetry == 0x9D` is the max value for now, so can we use that?

```cpp
template <>
struct magic_enum::customize::enum_range<::uwb::protocol::fira::UwbConfiguration::ParameterTag> {
  static constexpr int min = 0x80; // Current minimum value is UwbConfiguration::ParameterTag::FiraPhyVersion == 0x80
  static constexpr int max = 0x9D; // Current maximum value is UwbConfiguration::ParameterTag::MaxRrRetry == 0x9D
};
```

Reply:
Oh good point, that would be better. I'll make that change.

Reply:
I ended up adding this in NearObjectCli.hxx since NearObjectCli is where we use this. It may make sense to put this in UwbConfiguration instead, but I had issues with including `#include magic_enum.hpp`

Reply:
It's ok here for now 👍